persistence/info.go

```diff
@@ -4,13 +4,13 @@ import (
 	"os"
 )
 
-func NumBytesWritten(dir string, predicate func(os.FileInfo) bool) (uint64, error) {
+func NumBytesWritten(dir string, predicate func(os.FileInfo) bool) (uint64, int, error) {
```

Comment:
This function should be two functions: `NumBytesWritten` and `NumFilesWritten`. There is never a use case where both return values are used, only either the number of bytes or the number of files.

Both functions are also only called in a single place (in the methods of `DiskState`). Imo we can inline these there.

Reply:
Fixed.